hip/static/styles/pages/health_alert_index_page.scss

```diff
@@ -10,9 +10,23 @@
 
 .alert-table-hip {
   width: 100%;
+  border: 0;
+  border-spacing: 2px 0;
+  border-collapse: separate;
   th {
-    background-color: #323232;
-    color: #ffffff;
+    background-color: $table-header-grey;
```

Comment:
I think we should use $dark-gray to match the sidebar is-active link background. I put this in the PR for HP-48-50.